cmd/mindthegap/push/bundle/bundle.go

```diff
@@ -38,26 +38,26 @@ func NewCommand(out output.Output) *cobra.Command {
 	)
 
 	cmd := &cobra.Command{
-		Use:   "image-bundle",
-		Short: "Push images from an image bundle into an existing OCI registry",
+		Use:   bundleCmdName,
+		Short: "Push from a bundle into an existing OCI registry",
```

Comment:
Should this be like since multiple bundles are supportd:
```suggestion
		Short: "Push from bundles into an existing OCI registry",
```